README.md

```diff
@@ -87,12 +87,13 @@ You can use any valid HEX color or pick from a predefined set of named colors (`
 
 The following styles are available (`flat` is the default).
 
-| style | demo |
-| ----- | ---- |
-| `flat` | ![](https://img.shields.io/static/v1?label=Profile+views&message=1234567890&color=007ec6&style=flat) |
-| `flat-square` | ![](https://img.shields.io/static/v1?label=Profile+views&message=1234567890&color=007ec6&style=flat-square) |
-| `plastic` | ![](https://img.shields.io/static/v1?label=Profile+views&message=1234567890&color=007ec6&style=plastic) |
+| style           | demo                                                                                                          |
+|-----------------|---------------------------------------------------------------------------------------------------------------|
+| `flat`          | ![](https://img.shields.io/static/v1?label=Profile+views&message=1234567890&color=007ec6&style=flat)          |
+| `flat-square`   | ![](https://img.shields.io/static/v1?label=Profile+views&message=1234567890&color=007ec6&style=flat-square)   |
+| `plastic`       | ![](https://img.shields.io/static/v1?label=Profile+views&message=1234567890&color=007ec6&style=plastic)       |
 | `for-the-badge` | ![](https://img.shields.io/static/v1?label=Profile+views&message=1234567890&color=007ec6&style=for-the-badge) |
+| `pixel`         | this is "invisible" mode for counter. Use it when you want to have counter without of displaying of it        |
```

Comment:
I'd prefer to not add spaces between columns. Otherwise it adds many changes to other rows.

Reply:
Ok. I'll revert it today later

Reply:
Hi @antonkomarev,

Fixed. Could you have a look?